### DISCUSSION TITLE
Prevent of value of "download-queue-size" in settings.json from getting overwritten

### DISCUSSION THREAD

**Discussion by ATL-Flaneur** [haugene/docker-transmission-openvpn#2215] [Q&A] [answered]
How do I make changes to `download-queue-size` in settings.json stick? I'm trying to increase the value from the default of 5, but when I re-launch the container the value I enter gets overwritten with 5.

**Comment by pkishino** [comment, marked as answer]
How are you changing the value? In the UI? Or in the settings file directly?
Easiest is to change in the UI, it persists to file. If you change in the file directly you need to stop the container first before changing.
Also check you aren’t seeing the value in your run command

**Reply by dynamiccarrots** [reply]
Would this be possible to add to the docker config, this is something that I am sure a lot of people would want to change.

**Reply by pkishino** [reply]
Yes, course you can add the value (like any of the transmission settings) to your docker config if you want

**Comment by ATL-Flaneur** [comment]
Thanks @pkishino! My mistake was changing the value in the JSON file while the container was still running, then restarting the container. Dumb move on my part. I shut down the container, made the change, and got the results I wanted.

AFAIK this value isn't changeable via the default web UI. You need to edit the JSON file or use an alternative client like [Transmission GUI](https://sourceforge.net/projects/transgui/).
